packages/orchestration/test/facade.test.ts

```diff
@@ -42,6 +48,7 @@ test('chain info', async t => {
   t.deepEqual(await result.getChainInfo(), mockChainInfo);
 });
 
+// XXX what to do with this test now that chainInfos is the caller's responsibility?
```

Comment:
@turadg what to do with this test now that chainInfos is the caller's responsibility?

Reply:
hmm yeah, the main point of it was the name registry behavior..

My first thought it so leave it around as a stub for other "contract upgrade" assertions, but if someone shows up to the test as-is they're going to be confused why it exists. So I think it best be removed.

Reply:
I pruned it to a `test.todo()`